### ISSUE TITLE
A question about hyperparameters of training on multiwoz2.0

### ISSUE THREAD

**Issue by SparkJiao** [SKTBrain/SUMBT#2]
Hi, very appericate to your contribution and I have two small questions.
1. I saw the `num_train_epochs` in the shell is `300`. How many epochs will the training process long actually?
2. How large is the memory of a single GPU you used? I have tried to train it on a TITAN XP but met OOM and I then set the training batch size as 2.
Thanks you very much!

**Comment by hwaranlee** [comment]
@SparkJiao Thanks for your questions.

1. The number of epochs depends on the warmup proportion. And the training early stops 40~100 epochs in WOZ case and ~300 epochs for MultiWOZ.

2. We used Tesla P40 (24GB) and Tesla V100 (34GB).

**Comment by SparkJiao** [comment]
OK， I got it. Thank you very much!
